### DISCUSSION TITLE
Supported filetypes

### DISCUSSION THREAD

**Discussion by thnewlands** [Voltstro-Studios/UnityWebBrowser#79] [General]
I've found so far the browser handles a really broad range of media. I haven't been able to figure out how to load mp4 and pdf files yet though. Is this a limitation with CEF?

**Comment by thnewlands** [comment]
Answering my own question (yes): https://github.com/SafeExamBrowser/seb-win-refactoring/issues/202

**Reply by thnewlands** [reply]
here's a ref from the source: https://github.com/cefsharp/CefSharp/wiki/General-Usage#multimedia-audiovideo

**Comment by Voltstro** [comment]
The builds provided by CEF lack any of the proprietary codecs. You would have to build CEF your self with codecs included, but then there is licensing to deal with (which I have no clue on how that is handled).

**Reply by thnewlands** [reply]
That's a bummer!! Seems like even paid packages for browsing the web in unity don't try to solve the problem. I was enjoying opening files on my desktop with the browser. I guess I can just convert them to WebM.
